.env.example

```diff
@@ -14,4 +14,10 @@ DATABASE_URL=
 APP_URL=http://localhost:4002
 
 SVIX_URL=https://api.eu.svix.com
-SVIX_API_KEY=
\ No newline at end of file
+SVIX_API_KEY=
+
+GITHUB_ID=
```

Comment:
Rename this to GITHUB_CLIENT_ID and Rename this to GITHUB_CLIENT_SECRET

Reply:
Resolved.